embedded-aerospike-enterprise/src/test/java/com/playtika/testcontainers/aerospike/enterprise/ValidateEnterpriseAerospikeBootstrapConfigurationTest.java

```diff
@@ -20,13 +20,13 @@ void failOnNonEnterpriseImage() {
 
     @Test
     void failOnUnsuitableEnterpriseImageVersion() {
-        contextRunner.withPropertyValues("embedded.aerospike.dockerImage=aerospike/aerospike-server-enterprise:6.1.0.16_1")
+        contextRunner.withPropertyValues("embedded.aerospike.dockerImage=aerospike/aerospike-server-enterprise:6.1.0.16")
                 .run(context -> assertThat(context).hasFailed());
     }
 
     @Test
     void skipValidation() {
-        contextRunner.withPropertyValues("embedded.aerospike.dockerImage=aerospike-server:6.1.0.16_1",
+        contextRunner.withPropertyValues("embedded.aerospike.dockerImage=aerospike-server:6.1.0.16",
```

Comment:
why not 16_1?

Reply:
it just checks version format, no matter